contracts/contracts/vault/VaultAdmin.sol

```diff
@@ -343,6 +343,45 @@ contract VaultAdmin is VaultStorage {
         emit AssetSupported(_asset);
     }
 
+    /**
+     * @notice Remove a supported asset from the Vault
+     * @param _asset Address of asset
+     */
+    function removeAsset(address _asset) external onlyGovernor {
+        require(assets[_asset].isSupported, "Asset not supported");
+        require(
+            IVault(address(this)).checkBalance(_asset) <= 1e6,
```

Comment:
I saw the original version had a check for zero balance, considering you might have dust on multiple strategies as well 1e6 might be a little low, and would advise for 10e12 which is even for a coin like ETH not worth more than a cent.

Reply:
Sounds good to me, making that change

Reply:
Done